packages/react/src/components/header/components/navigationLink/navigationLinkDropdown/NavigationLinkDropdown.module.scss

```diff
@@ -0,0 +1,87 @@
+.navigationLinkDropdownContainer {
+  outline: none;
+}
+
+.chevron {
+  transition: transform 0.2s ease-in;
+}
+
+.chevronOpen {
+  transform: rotate(180deg);
+}
+
+.hidden {
+  display: none;
+}
+
+.dropdownMenu {
+  background-color: var(--color-white);
+  border: 1px solid var(--color-black-20);
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  min-width: 280px;
+  padding: 0;
+  position: absolute;
+  left: -10px;
+  top: 100%;
+  z-index: 1;
+
+  .dropdownLink {
+    font-size: var(--fontsize-body-l);
+    margin: 0;
+    padding: 10px var(--spacing-s) 14px var(--spacing-s);
+  }
+
+  .activeLink::before {
+    border-left: 4px solid var(--color-black);
+    padding-left: var(--spacing-xs);
+  }
+
+  .dropdownLink::before {
+    content: '';
+    height: 100%;
+    margin: 0;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 0;
+  }
+
+  .dropdownLink:hover::before {
+    border-left: 4px solid var(--color-black);
+    transition: border-left 100ms ease-in;
+  }
+}
+
+.left {
+  left: -282px;
+  top: -1px;
+}
+
+.right {
+  left: 282px;
+  top: -1px;
+}
```

Comment:
I was able to fix the position with wider than 280px parent menus:

```suggestion
.left {
  left: auto;
  right: calc(100% + 2px);
  top: -1px;
}

.right {
  left: calc(100% + 2px);
  right: auto;
  top: -1px;
}
```
<img width="894" alt="Screenshot 2022-12-29 at 17 06 09" src="https://user-images.githubusercontent.com/1610860/209971993-759114c4-8943-40bf-9d21-6823bc53f6d0.png">
 
to:
<img width="1162" alt="Screenshot 2022-12-29 at 17 05 52" src="https://user-images.githubusercontent.com/1610860/209971949-1226d800-2127-4e40-9b91-5bfa987b0e2a.png">

Reply:
Chef's kiss!